entities/weapons/arrest_stick/shared.lua

```diff
@@ -53,6 +53,7 @@ function SWEP:Deploy()
     return BaseClass.Deploy(self)
 end
 
+local distance = SWEP.stickRange * SWEP.stickRange
```

Comment:
This will cache an incorrect value if arrest_stick is inherited and the stickRange is changed. If you want to optimise it, do
```
local range = self.stickRange
if not IsValid(ent) or (self:GetOwner():EyePos():DistToSqr(ent:GetPos()) > range * range) or not ent:IsPlayer() then
```

Reply:
Then the optimisation is less significant, but okay :)